cluster/ci/monitoring/mixins/config.libsonnet

```diff
@@ -47,6 +47,11 @@
         channel: '#forum-os-dev-services',
         notify: 'openshift-app-services',
       },
+      'OLM-rh-operators': {
+        team: 'OLM',
+        channel: '#operator-test',
```

Comment:
operator-test is enough for monitoring team and we will just watch that channel, if we need to we can always adjust in the future. Thanks!